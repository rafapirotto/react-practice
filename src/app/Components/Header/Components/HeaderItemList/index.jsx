import React from 'react';

import HeaderItem from '../HeaderItem';
import { MY_LIST_ROUTE } from '../../../../routes';

const HeaderItemList = () => (
  <div className="flex ml-32 items-center flex-wrap">
    <HeaderItem title="Home" styles="mx-6 font-normal" id="home-btn" />
    <HeaderItem title="Movies" styles="mx-6" id="movies-btn" />
    <HeaderItem title="Series" styles="mx-6" id="series-btn" />
    <HeaderItem title="Recently Added" styles="mx-6" id="recently-added-btn" />
    <HeaderItem title="My List" styles="mx-6" to={MY_LIST_ROUTE} id="my-list-btn" />
  </div>
);

export default HeaderItemList;
