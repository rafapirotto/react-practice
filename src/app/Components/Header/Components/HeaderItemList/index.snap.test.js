import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import HeaderItemList from '.';

test('It matches the snapshot', () => {
  const component = renderer.create(
    <Router>
      <HeaderItemList />
    </Router>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
