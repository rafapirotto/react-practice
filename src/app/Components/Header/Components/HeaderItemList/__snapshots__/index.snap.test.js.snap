// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<div
  className="flex ml-32 items-center flex-wrap"
>
  <div
    className="mx-6 font-normal"
    id="home-btn"
  >
    <a
      href="/dashboard"
      onClick={[Function]}
    >
      Home
    </a>
  </div>
  <div
    className="mx-6"
    id="movies-btn"
  >
    <a
      href="/dashboard"
      onClick={[Function]}
    >
      Movies
    </a>
  </div>
  <div
    className="mx-6"
    id="series-btn"
  >
    <a
      href="/dashboard"
      onClick={[Function]}
    >
      Series
    </a>
  </div>
  <div
    className="mx-6"
    id="recently-added-btn"
  >
    <a
      href="/dashboard"
      onClick={[Function]}
    >
      Recently Added
    </a>
  </div>
  <div
    className="mx-6"
    id="my-list-btn"
  >
    <a
      href="/dashboard/my-list"
      onClick={[Function]}
    >
      My List
    </a>
  </div>
</div>
`;
